### ISSUE TITLE
NetStandard library getting installed in Android and iOS

### ISSUE THREAD

**Issue by alexrainman** [aritchie/userdialogs#537]
## Please check all of the platforms you are having the issue on

 - [*] iOS
 - [*] Android
 - [] UWP
 - [] NET Standard 

## Version of OS(s) listed above with issue

Xamarin.Android.Support.* 26.1.0.1
iOS 11.4

## Version of ACR UserDialogs Library

7.0.x

## Expected Behaviour

Install platform specific package

## Actual Behavior

Net Standard libraries being installed in platform specific projects

IS THERE A WAY TO AVOID IT?

**Comment by aritchie** [comment]
Works fine for me.  Make sure you have android 8+ as target.  I usually install in the platforms as well (even though packagereference is supposed to fix this)

**Comment by alexrainman** [comment]
The plugin works, as expected :) Just having those extra Net Standard packages installed on Android and iOS. Is there a way to fix that? I am using Visual Studio for Mac.

**Comment by alexrainman** [comment]
I noticed the are like ghost packages. I can remove them after installing version 7.0.x

Same was happening with Json.Net and it was fixed in version 11.

**Comment by aritchie** [comment]
There isn’t anything I can do since it is some sort of Nuget bug.  It should uninstall the old version first.  For some reason, it leaves behind Acr.UserDialogs.Abstractions which doesn’t exist in v7 due to how I use multitargeting.  I’ll likely just add something to the faq section

**Comment by alexrainman** [comment]
I will follow your advice and install a fresh copy of the plugin and remove any ghost package. I found other plugins with same issue so it is not only yours.

I also updated my plugins to multi targeting and net standard 2, including my Google Analytics plugin and my version of modernhttpclient which includes some cool features and updates like setting native handler timeout, support for self signed certificates, working cookie handler, uwp support, OkHttp3, net standard 2 and major bug fixes.

Thanks for your quick response and this great and very useful plugin.
